### PR TITLE
fix(DB/Loot): Syndicate emblems in junkboxes

### PR DESCRIPTION
**Pull request by BeefSteaker**
## Changes Proposed:
-  Remove Syndicate emblems from Battered Junkbox and Worn Junkbox

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11360

## Tests Performed:

- Checked Kiera loot table after running SQL query
- opened a few boxes and no obvious issues, but low sample size

## How to Test the Changes:

1. .setskill 633 300 300
2. .additem 16882
3. .additem 16883

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by temperrr — APPROVED** [review]

File: data/sql/updates/pending_db_world/removeloot.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+-- Remove Syndicate emblems from Battered Junkbox and Worn Junkbox
+DELETE FROM `item_loot_template` WHERE (`Entry` = 16882 or `Entry` = 16883) AND (`Item` IN (17124));
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/removeloot.sql:2]: Use `Entry` IN instead of or expression

**Review by Kitzunu — CHANGES_REQUESTED** [review]
Correct the statement above

**Comment by BeefSteaker** [comment]
Fixed

File: data/sql/updates/pending_db_world/removeloot.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+-- Remove Syndicate emblems from Battered Junkbox and Worn Junkbox
+DELETE FROM `item_loot_template` WHERE `Entry` in (16882, 16883) AND (`Item` IN (17124));
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/removeloot.sql:2]:
```suggestion
DELETE FROM `item_loot_template` WHERE `Entry` IN (16882, 16883) AND (`Item` IN (17124));
```

Comment (BeefSteaker) [reply data/sql/updates/pending_db_world/removeloot.sql]: Thanks

**Review by Kitzunu — APPROVED** [review]

**Review by Nefertumm — APPROVED** [review]

**Pull request merged**